daemons/gptp/common/ieee1588port.cpp

```diff
@@ -712,7 +712,7 @@ void IEEE1588Port::processEvent(Event e)
 		break;
 
 	case LINKUP:
-		haltPdelay(false);
+		stopPDelay();
```

Comment:
There should be a TODO here, right? Modify the code so that gptp doesn't even start the pdelay thread until the link is ready.